core/templates/pages/classroom-page/classroom-page.controller.ts

```diff
@@ -54,7 +54,12 @@ angular.module('oppia').directive('classroomPage', [
             ctrl.bannerImageFileUrl = UrlInterpolationService.getStaticImageUrl(
               '/splash/books.svg');
 
-            PageTitleService.setPageTitle(ctrl.classroomName + ' - Oppia');
+            ctrl.classroomDisplayName = (
+              ctrl.classroomName.charAt(0).toUpperCase() +
+              ctrl.classroomName.slice(1));
+
+            PageTitleService.setPageTitle(ctrl.classroomDisplayName +
+                ' Classroom | Oppia');
```

Comment:
Indent by 2, not 4.

Also, on previous line, break after '('.

**In general, please be careful to follow the existing [coding style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide).**

Reply:
Done, sorry!
I'm not sure why the linter didn't trigger here. It usually shows a warning it there is no break after `(`.

Reply:
Hi @Hudda -- do you know why this happened?

Reply:
@seanlip I don't think we have this check implemented, but can be added by adding `"function-paren-newline": ["error", "multiline"]` in `.eslintrc` file. But then we also need to add newline before ')'. If it's fine with you then I will add this check.

Reply:
Yep, I'm fine with this. Thanks for investigating, @Hudda!